extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetUtils.java

```diff
@@ -51,7 +44,9 @@ public static boolean isVisibleParquetURI(final URI uri) {
         if (!path.endsWith(PARQUET_FILE_EXTENSION)) {
             return false;
         }
-        // Look for hidden directories or files in the path
-        return !HIDDEN_FILE_PATTERN.matcher(path).find();
+        if (FILE_URI_SCHEME.equals(uri.getScheme())) {
+            return !HIDDEN_FILE_PATTERN.matcher(path).find();
```

Comment:
Given we know it's a File, should we prefer to use `java.nio.file.Path#getParent` and `Path#getFileName` to check in a loop? At a minimum, it's easier for me to audit that as compared to Regex (and, it's _probably_ faster?).